converter/src/main/java/gov/cms/qpp/conversion/decode/QualityMeasureIdDecoder.java

```diff
@@ -39,7 +39,7 @@ protected DecodeResult internalDecode(Element element, Node thisNode) {
 		List<String> measureGuids = getMeasureGuid(element);
 
 		measureGuids.forEach(measureGuid ->
-			thisNode.putValue(MEASURE_ID, measureGuid, false));
+			thisNode.putValue(MEASURE_ID, measureGuid.toLowerCase(), false));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1449)